addons/mail/mail_followers.py

```diff
@@ -217,7 +220,14 @@ def _notify_email(self, cr, uid, ids, message_id, force_send=False, user_signatu
             }
             mail_values.update(custom_values)
             email_ids.append(self.pool.get('mail.mail').create(cr, uid, mail_values, context=context))
-        if force_send and len(chunks) < 2:  # for more than 50 followers, use the queue system
+        # NOTE:
+        #   1. for more than 50 followers, use the queue system
+        #   2. do not send emails immediately if the registry is not loaded,
+        #      to prevent sending email during a simple update of the database
+        #      using the command-line.
+        if force_send and len(chunks) < 2 and \
+           (not self.pool._init or
```

Comment:
When the wrapped condition ends up aligned with the contents of the `if` block, please indent the condition by one extra level to make the block contents stand out :-)